### PR TITLE
fix(chat): disable auto redirect from chat to friend list on first visit

### PR DESCRIPTION
**Pull request by AlexZakablukov**
**What this PR does** 📖
disable auto redirect from chat to friend list on first visit

**Which issue(s) this PR fixes** 🔨
[AP-1643](https://satellite-im.atlassian.net/browse/AP-1643)

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Review by Jekrimo — APPROVED** [review]

File: store/friends/actions.ts [hunk 1]
```diff
@@ -42,15 +42,15 @@ export default {
       { root: true },
     )
 
+    await dispatch('fetchFriends', {})
+    await dispatch('fetchFriendRequests', {})
+    await dispatch('subscribeToFriendsEvents', {})
```
Comment (molimauro) [inline store/friends/actions.ts:47]: Do not await consequently, just use `Promise.all`

File: store/friends/actions.ts [hunk 2]
```diff
@@ -108,9 +108,11 @@ export default {
...
     // Concat incoming and outgoing friends into a single array
     // and fetch user info for each friend
-    incoming
-      .concat(outgoing)
-      .forEach((friendData) => dispatch('fetchFriendDetails', friendData))
+    const friendData = incoming.concat(outgoing)
+
+    for (let i = 0; i < friendData.length; i++) {
+      await dispatch('fetchFriendDetails', friendData[i])
+    }
```
Comment (molimauro) [inline store/friends/actions.ts:115]: same

Comment (AlexZakablukov) [reply store/friends/actions.ts]: fixed

File: store/friends/actions.ts (hunk 1 above)
Comment (AlexZakablukov) [reply store/friends/actions.ts]: fixed

**Review by molimauro — APPROVED** [review]

**Review by phillsatellite — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Pull request merged**